### PR TITLE
Terraform scripts and documentation to deploy ECS Task Definition for PDS Validate Tool

### PR DESCRIPTION
**Pull request by ramesh-maddegoda**
## 🗒️ Summary
This pull request adds terraform scripts and documentation to deploy ECS Task Definition for PDS Validate Tool.

## ♻️ Related Issues
NASA-PDS/devops#42

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+cd validate/terraform
+```
+
+3. Set following environment variables in terminal window
+    - AWS_ACCESS_KEY_ID
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_SESSION_TOKEN
+    - AWS_DEFAULT_REGION
+
+4. Open the `terraform.tfvars` file locally and enter the value for following variables. Ensure these values match with your AWS Setup. Most of the below values can be obtained by the system admin team of your AWS account.
```
Comment (jordanpadams) [inline terraform/README.md]: @ramesh-maddegoda this works for now, but I think we should try to figure out a more robust solution using AWS Secrets per the same issue @sjoshi-jpl is encountering with his CI/CD work. We can revise this once we come up with a better solution.

File: terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+       "portMappings": [],
+       "essential": true,
+       "environment": [],
+       "mountPoints": [],
+       "volumesFrom": [],
+       "logConfiguration": {
+           "logDriver": "awslogs",
+           "options": {
+               "awslogs-group": "/ecs/pds-validate",
+               "awslogs-region": "us-west-2",
```
Comment (jordanpadams) [inline terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json]: This seems like something we may want to change at some point, but not a huge deal...

**Review by jordanpadams — APPROVED** [review]

**Comment by jordanpadams** [comment]
@sjoshi-jpl can you review this when you have a chance?

**Review by nutjob4life — APPROVED** [review]

File: terraform/README.md (hunk 1 above)
Comment (sjoshi-jpl) [reply terraform/README.md]: @ramesh-maddegoda @jordanpadams agree with using AWS Secrets approach but that will be something all TF workflows will need to use once we have CI/CD established in MCP. So for now I think this look good.

File: terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json (hunk 2 above)
Comment (sjoshi-jpl) [reply terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json]: @ramesh-maddegoda can these be passed in as env variables during workflow execution?

Comment (ramesh-maddegoda) [reply terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json]: @sjoshi-jpl and @jordanpadams , I externalized these values as variables that we pass in a tfvars file (same approach we used for Nucleus)

Comment (sjoshi-jpl) [reply terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json]: Looks good now. Approved!

**Review by sjoshi-jpl — APPROVED** [review]

**Pull request merged**